src/main/java/mate/academy/spring/service/impl/AuthenticationServiceImpl.java

```diff
@@ -18,10 +21,13 @@ public AuthenticationServiceImpl(UserService userService,
     }
 
     @Override
-    public User register(String email, String password) {
+    public User register(String email, String password, Role role) {
```

Comment:
```suggestion
    public User register(String email, String password) {
```
![image](https://user-images.githubusercontent.com/20326858/214271871-7e0d1c6f-e6c7-4e0b-9e5b-4a5de66f7c00.png)